ucte/ucte-converter/src/main/java/com/powsybl/ucte/converter/UcteImporter.java

```diff
@@ -257,6 +247,23 @@ private static void createDanglingLine(UcteLine ucteLine, boolean connected,
                 .setQ0(q0)
                 .setUcteXnodeCode(xnode.getCode().toString())
                 .add();
+        dl.newGeneration()
+                .setTargetP(-targetP)
+                .setTargetQ(-targetQ)
+                .add();
```

Comment:
You should create a generation only if necessary: i.e there is a voltage setpoint or a power generation?

Reply:
@annetill Are these values used in loadflow even when there is no voltage setpoint and/or power generation?

Reply:
If the generator is regulating voltage, I use the voltage setpoint, otherwise I use the generator targetQ. From CGMES files, I will most of the time use only P0 and Q0. From UCTE file, I will most of the time have both load part and generation part, as it is described in the input data.